dotcom-rendering/src/web/server/blocksToHtml.tsx

```diff
@@ -0,0 +1,71 @@
+import { renderToString } from 'react-dom/server';
+import { buildAdTargeting } from '@root/src/lib/ad-targeting';
+import { decideDesign } from '../lib/decideDesign';
+import { decideDisplay } from '../lib/decideDisplay';
+import { decideTheme } from '../lib/decideTheme';
+import { LiveBlogRenderer } from '../lib/LiveBlogRenderer';
+
+/**
+ * blocksToHtml is used by the /Blocks endpoint as part of keeping liveblogs live
+ * It takes an array of json blocks and returns the resulting html string
+ *
+ * @param blocks Block[]
+ * @param format CAPIFormat
+ * @param host string
+ * @param pageId string
+ * @param webTitle string
+ * @param ajaxUrl string
+ * @param isAdFreeUser boolean
+ * @param isSensitive boolean
+ * @param edition string
+ * @param section string
+ * @param sharedAdTargeting Record<string, unknown>
+ * @param adUnit string
+ * @param videoDuration: number | undefined
+ * @returns string (the html)
```

Comment:
This doesn’t add anything if you already use TypeScript. I also think the syntax is slightly off, the type should come first and be inside curly brackets.

It’s worth defining param only if you want to add some information about them.

Reply:
You're right. These really don't add anything. Removed